apps/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -96,6 +104,10 @@ export abstract class BaseInstallAction extends BaseRushAction {
       }
     }
 
+    if (this._maxInstallAttempts.value && this._maxInstallAttempts.value < 1) {
+      throw new Error(`The value of "${this._maxInstallAttempts.longName}" must be positive.`);
```

Comment:
```suggestion
      throw new Error(`The value of "${this._maxInstallAttempts.longName}" must be positive and nonzero.`);
```

Reply:
positive implies non-zero, right?

Reply:
Zero is generally considered either both positive and negative or neither positive nor negative: https://math.stackexchange.com/questions/26705/is-zero-positive-or-negative